spec/factory_specs/course_spec.rb

```diff
@@ -1,18 +1,31 @@
 require 'rails_helper'
 
-describe "Course Factory" do
-  let(:course) { create(:course) }
+describe "Course factory" do
+  subject { create(:course) }
 
-  it "created course" do
-    expect(course).to be_instance_of(Course)
-    expect(course).to be_valid
-  end
+  it { should be_instance_of(Course) }
```

Comment:
Do we really need to test for this?

Reply:
It may be pedantic, but I like having tests that prove the factories work. Also, it's CHEAP!